conductor/src/main.rs

```diff
@@ -420,6 +450,21 @@ async fn run(metrics: CustomMetrics) -> Result<(), ConductorError> {
                 info!("{}: Deleting cloudformation stack", read_msg.msg_id);
                 delete_cloudformation(aws_region.clone(), &namespace).await?;
```

Comment:
Nick mentioned need to handle not doing this line unless AWS backups were enabled. Maybe by skipping it unless it's AWS